controllers/etcd_controller_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package controllers_test
+package controllers
```

Comment:
See comment below.

Reply:
lets keep it the same package unless we cannot go ahead this way.